### PR TITLE
[Abort] Cross Attention Zipformer

### PR DESCRIPTION
**Pull request by yfyeung**

### CODE REVIEW

File: egs/librispeech/ASR/zipformer/zipformer.py [hunk 1]
```diff
@@ -1434,7 +1497,9 @@ def __init__(
         # dividing it between the query and key.   Note: this module is intended
         # to be used with the ScaledAdam optimizer; with most other optimizers,
         # it would be necessary to apply the scaling factor in the forward function.
-        self.in_proj = ScaledLinear(embed_dim, in_proj_dim, bias=True,
+        self.in_proj_a = ScaledLinear(embed_dim, in_proj_dim, bias=True,
```
Comment (yaozengwei) [inline egs/librispeech/ASR/zipformer/zipformer.py:1500]: `in_proj_a` and `in_proj_b` should have their own in_proj_dim:
* in_proj_a: (query_head_dim + pos_head_dim) * num_heads
* in_proj_b: key_head_dim * num_heads

File: egs/librispeech/ASR/zipformer/zipformer.py [hunk 2]
```diff
@@ -1499,7 +1567,7 @@ def forward(
 
         # self-attention
         q = x[...,0:query_dim]
-        k = x[...,query_dim:2*query_dim]
+        k = y[...,query_dim:2*query_dim]
```
Comment (yaozengwei) [inline egs/librispeech/ASR/zipformer/zipformer.py:1570]: The slice index would need to be changed as in above comment

Comment (yfyeung) [reply egs/librispeech/ASR/zipformer/zipformer.py]: OK

File: egs/librispeech/ASR/zipformer/zipformer.py (hunk 1 above)
Comment (yfyeung) [reply egs/librispeech/ASR/zipformer/zipformer.py]: OK

**Pull request closed without merging**